api/app/Models/User.php

```diff
@@ -475,6 +476,45 @@ public function getPositionDuration()
         return null;
     }
 
+    public function getPriority()
```

Comment:
I'd recommend deriving this from the actual business data columns rather than the weight column.  Ref: https://github.com/GCTC-NTGC/gc-digital-talent/issues/8668

Reply:
okay, so are you suggesting something like:

```php
$priority = [];

if($this->has_priority_entitlement) {
  $priority[] = 'Priority entitlement'; 
}

// etc.

return implode(', ', $priority);
```

Reply:
Updated in [update priority status column value](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9988/commits/5784d9472d15e0c5ab45cd50df48bea52c2d0122)